allennlp/data/fields/text_field.py

```diff
@@ -88,29 +94,32 @@ def get_padding_lengths(self) -> Dict[str, int]:
 
         # Each indexer can return a different sequence length, and for indexers that return
         # multiple arrays each can have a different length.  We'll keep track of them here.
-        for indexer_name, indexer in self._token_indexers.items():
+        for token_index, indexer in self._token_indexers.items():
             indexer_lengths = {}
 
-            for indexed_tokens_key in self._indexer_name_to_indexed_token[indexer_name]:
+            for indexed_tokens_key in self._indexer_name_to_indexed_token[token_index]:
                 # This is a list of dicts, one for each token in the field.
                 token_lengths = [indexer.get_padding_lengths(token)
                                  for token in self._indexed_tokens[indexed_tokens_key]]
-            if not token_lengths:
```

Comment:
Is this a bug?  I'm trying to understand why this PR is changing this indentation, and it looks to me like it's fixing an unrelated bug.  Is that right?  (If so, it's fine, I was just confused at seeing this change here.)

Reply:
Yes. It tried to fix another bug. Sorry for the confusion.